pages/linux/enum4linux.md

```diff
@@ -0,0 +1,24 @@
+# enum4linux
+
+> Enum4linux is a tool for enumerating information from Windows and Samba systems.
+> It attempts to offer similar functionality to enum.exe formerly available from www.bindview.com.
+
+- Try to enumerate using all methods:
+
+`enum4linux -a remote_host`
+
+- Enumerate using login:
+
+`enum4liux -u user_name -p password remote_host`
```

Comment:
Does it prompt for a password if you omit it? That would be more secure.

Also, token syntax?
```suggestion
`enum4liux -u {{user_name}} -p {{password}} {{remote_host}}`
```

Finally, are there any long forms of these options? We tend to prefer those (same goes for the examples further down.